### PR TITLE
webui: add helm charts for 33.x and future releases

### PR DESCRIPTION
**Pull request by maany**
Modifies the existing `rucio-webui` helm chart to incorporate the NextJS application

### CODE REVIEW

File: charts/rucio-webui/README.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+
+##  Data Management for science in the Big Data era.
+
+Rucio is a software framework that provides functionality to organize, manage, and access large volumes of scientific data using customisable policies. The data can be spread across globally distributed locations and across heterogeneous data centers, uniting different storage and network technologies as a single federated entity. Rucio offers advanced features such as distributed data recovery or adaptive replication, and is highly scalable, modular, and extensible. Rucio has been originally developed to meet the requirements of the high-energy physics experiment ATLAS, and is continuously extended to support LHC experiments and other diverse scientific communities.
+
+## QuickStart
+
+Add the Rucio Helm repository to your local Helm installation:
+```bash
+helm repo add rucio https://rucio.github.io/helm-charts
```
Comment (rdimaio) [inline charts/rucio-webui/README.md:11]: this link returns 404 at the moment https://rucio.github.io/helm-charts - not an issue but just wanted to point it out

File: charts/rucio-webui/README.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+  port: 443
+  targetPort: 443
+  protocol: TCP
+  name: https
+```
+
+Furthermore, you can also change the service type depending on how you want to expose the WebUI. For example, to expose the WebUI using a `LoadBalancer` service, you would set the `service.type` parameter to `LoadBalancer`. To expose the WebUI using a `NodePort` service, you would set the `service.type` parameter to `NodePort` and the `service.nodePort` parameter to the desired port.
+
+## Ingress
+If you want to use and ingress controller to expose the servers you will have to
```
Comment (rdimaio) [inline charts/rucio-webui/README.md:121]:
```suggestion
If you want to use an ingress controller to expose the servers you will have to
```

**Review by rdimaio — APPROVED** [review]

**Comment by bari12** [comment]
@maany I just update the main branch for the Rucio 34 release. Please update this PR

**Comment by bari12** [comment]
Still conflicts :-)

File: charts/rucio-webui/README.md (hunk 1 above)
Comment (maany) [reply charts/rucio-webui/README.md:11]: It's okay. The file of interest is not index.html but index.yaml
https://rucio.github.io/helm-charts/index.yaml

**Comment by maany** [comment]
@bari12 all set :)

**Pull request merged**